.travis.yml

```diff
@@ -0,0 +1,21 @@
+language: ruby
+cache: bundler
+rvm:
+  - 2.4.1
+script:
+  - bundle install
+  - bundle exec rake db:create --all
+  - bundle exec rake db:migrate
+  - bundle exec rake db:test:prepare
+  - bundle exec rake
+services:
+  - postgresql
+addons:
+  chrome: stable
+install:
```

Comment:
Doesn't make any difference to Travis but IMO it makes sense to order these in the order in which they're actually executed.

So I'd move this step above before the `script` and `before_script`